recipes/gala/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "gala" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "63a844c6d6c6e6771426472b85a36624cb85e188d08b4c4593a672b4969a65d3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py < 35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy x.x
```

Comment:
To further restrict `numpy` one can tweak this as follows.

```yaml
    - numpy x.x
    - numpy >=1.7
```

This would then be repeated in `run` too.